doc/integration_docs/4_builtinpackages.md

```diff
@@ -33,11 +36,11 @@ For internal ADSK packages intended for `Built-In Packages` inclusion - the curr
 
 Using a workaround it's possible to manually create (and even publish) packages with culture subdirectories in the /bin folder of a package.
 
-Create the culture specific subdirectories you require under the packages's `/bin` folder manually. 
+First create the culture specific subdirectories you require under the packages's `/bin` folder manually. 
 
-If for some reason, the package needs to also be published to the package manager then you must first publish a version of the package that is missing these culture subfolders - then publish a new version of the package using the DynamoUI `publish package version`. The new version upload in Dynamo should not delete your the folders and files under`/bin` which you have added manually using the windows file explorer. The package upload process in Dynamo will be updated to deal with the requirements for localized files in the future.
+If for some reason, the package needs to also be published to the package manager then you must first publish a version of the package that is missing these culture subdirectories - then publish a new version of the package using the DynamoUI `publish package version`. The new version upload in Dynamo should not delete your the folders and files under`/bin` which you have added manually using the windows file explorer. The package upload process in Dynamo will be updated to deal with the requirements for localized files in the future.
```

Comment:
There's an additional `the` in the third line; it should read: `should not delete your folders and files under`/bin`

Reply:
Not sure, I understand this, what is the new version required for?

Reply:
@aparajit-pratap this is part of a workaround for publishing a package that contains localized subdirectories inside the /bin folder.

1. You first need to publish a version without the subdirectories
2. Then create the directories with the resource binaries in the packages's bin folder.
3. Start dynamo and publish a new version of the package, dynamo will now keep the existing folders when publishing if the package is loaded from the standard package directory.

It's a fragile workaround, but it's the only one I know of at the moment.

AFAIK it was supposed to be a supported workflow in the visual refresh but time ran out.